samcli/local/docker/lambda_container.py

```diff
@@ -24,8 +24,6 @@ class LambdaContainer(Container):
     # The Volume Mount path for debug files in docker
     _DEBUGGER_VOLUME_MOUNT_PATH = "/tmp/lambci_debug_files"
     _DEFAULT_CONTAINER_DBG_GO_PATH = _DEBUGGER_VOLUME_MOUNT_PATH + "/dlv"
-    _RAPID_SOURCE_PATH = Path(__file__).parent.joinpath("..", "rapid").resolve()
-    _RAPID_DESTINATION_MOUNT = {"bind": "/var/rapid", "mode": "ro"}
     _GO_BOOTSTRAP_SOURCE_PATH = Path(__file__).parent.joinpath("..", "go-bootstrap").resolve()
```

Comment:
What are we going to do with these? Go debugging is going to have the same issue.